static/src/javascripts/bootstraps/enhanced/atoms.js

```diff
@@ -0,0 +1,71 @@
+// @flow
+
+import config from 'lib/config';
+import { services } from 'journalism/services';
+
+const bootstrapAtom = (atomMaker: AtomMaker, atomType: AtomType) => {
+    const atomBuilder = atomMaker[atomType].default(services);
+    Array.from(
+        document.querySelectorAll(`[data-atom-type='${atomType}']`)
+    ).forEach(atomDom => {
```

Comment:
Minor thing, but you could replace `Array.from(...)` with the delightfully terse:
```js
[...document.querySelectorAll]
```

Reply:
Indeed! There's a small perf penalty though, but I'm happy to use it.